pelias.json

```diff
@@ -47,24 +54,44 @@
       "countryCode": "ALL"
     },
     "openstreetmap": {
+      "download": [
+        {
+          "sourceURL": "https://s3.amazonaws.com/metro-extracts.mapzen.com/portland_oregon.osm.pbf"
+        }
+      ],
       "leveldbpath": "/tmp",
       "datapath": "/data/openstreetmap",
       "import": [{
-        "filename": "extract.osm.pbf"
+        "type": { "node": "osmnode", "way": "osmway" },
+        "filename": "portland_oregon.osm.pbf"
       }]
     },
     "openaddresses": {
       "datapath": "/data/openaddresses",
-      "files": [ "extract.csv" ]
```

Comment:
we use that variable to limit the download now, so it won't work if you specify a name that's not in OA